hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -903,7 +903,10 @@ private boolean matchSearchIndexWords(String propValue, String fieldValue) {
             return ImmutableSet.copyOf(texts);
         }
         Set<String> segments = this.textAnalyzer.segment(text);
+
+        // Add original words to segments, in order to can be match fully words
```

Comment:
for more specific like: "Add original text to segments at the insertion stage, in order to can match fully words at the query stage."